docs/input.md

```diff
@@ -219,6 +199,37 @@ description: |
     This is the second line
 ```
 
+#### `example`
+
+The `example` field is optional and can be used to reference a sample dataset
+for the given input, **within** the container. Data examples are a prime source 
+for your users to understand how inputs should look like and be formatted.
+
+```yaml
+example: /samples/input_name.csv
```

Comment:
The idea of an `example` field is great!  
We also always wanted to include exemplary data and parameters in the /in folder of a container but I think it is also a good idea to have be more clear about that and have that extra /samples folder.